backend/reservations/views/rooms.py

```diff
@@ -105,10 +105,12 @@ def room_list(request):
 
         logger.info(f"[+] Valid guest viewing rooms: {email}")
         rooms = Room.objects.all()
-        no_guest = list(filter(lambda x:x.guest==None, rooms))
-        for elem in no_guest:
-            elem.available=False
-            elem.save()
+
+        #no_guest = list(filter(lambda x:x.guest==None, rooms))
```

Comment:
Instead of commenting, can we just remove these lines?

Reply:
ja will pull now and add to next PR